mix.exs

```diff
@@ -53,7 +53,9 @@ defmodule Siwapp.MixProject do
       {:postgrex, ">= 0.0.0"},
       {:swoosh, "~> 1.3"},
       {:telemetry_metrics, "~> 0.6"},
-      {:telemetry_poller, "~> 1.0"}
+      {:telemetry_poller, "~> 1.0"},
+      {:contex, "~> 0.4.0"},
+      {:datix, "~> 0.1.1"}
```

Comment:
why this lib? You don't seem to use it, besides it's almost dead

Reply:
Oh, I did some trials with that. I forgot to remove it.